cognify/optimizer/core/flow.py

```diff
@@ -129,6 +129,7 @@ class OptConfig:
     param_save_path: str = field(default=None)
     frugal_eval_cost: bool = field(default=True)
     use_SH_allocation: bool = field(default=False)
+    patience: tuple[float,float,int] = field(default=(0.01,0.01,5))
```

Comment:
add more description to what each element in the tuple means. better with a macro, at least add a comment here and when you use it

Reply:
added doc string for this class